tools/Ensembl-REST/get_sequences/get_sequences.xml

```diff
@@ -24,10 +25,18 @@
             <option value="cdna">cDNA</option>
             <option value="protein">Protein</option>
         </param>
+        <param name="format_selector" type="select" label="Format" help="Format of the data">
+            <option value="fasta" selected="true">Fasta</option>
+            <option value="json">JSON</option>
+        </param>
     </inputs>
 
     <outputs>
-        <data name="output" format="json" label="$(tool.name) on ${on_string}" />
+        <data name="output" format="fasta" label="$(tool.name) on ${on_string}">
+            <change_format>
+                <when input="format_selector" value="json" format="json" />
+            </change_format>
+        </data>
     </outputs>
 
     <tests>
```

Comment:
Add
```
            <param name="format_selector" value="json" />
```
to the following test, otherwise it will break because default is `fasta` now.